runelite-client/src/main/java/net/runelite/client/plugins/bank/BankPlugin.java

```diff
@@ -278,7 +278,7 @@ else if (event.getScriptId() == ScriptID.BANKMAIN_SEARCH_REFRESH)
 			// vanilla only lays out the bank every 40 client ticks, so if the search input has changed,
 			// and the bank wasn't laid out this tick, lay it out early
 			final String inputText = client.getVar(VarClientStr.INPUT_TEXT);
-			if (searchString != inputText && client.getGameCycle() % 40 != 0)
+			if (!searchString.equals(inputText) && client.getGameCycle() % 40 != 0)
```

Comment:
I don't think this change is necessary? Is it required for this PR?

Reply:
I think this was a change intellij requested before I committed. I can change it back

Reply:
This is because intellij doesn't like using reference comparisons when comparing strings, but in this case it is fine.